wenet/squeezeformer/utils.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn as nn
+
+
+class ResidualModule(nn.Module):
+    """
```

Comment:
You can move it to encoder.py, `utils.py` is not a proper name.

Reply:
What if we just do it directly in `forward` function?

Reply:
I moved residual connection to encoder.py and removed the util.py.